proknow-sdk/Scorecard/ScorecardTemplates.cs

```diff
@@ -95,13 +104,15 @@ public async Task DeleteAsync(string scorecardTemplateId)
         /// Finds a scorecard template asynchronously based on a predicate
         /// </summary>
         /// <param name="predicate">The predicate for the search</param>
+        /// <param name="workspaceID">The ProKnow ID or name of the workspace or null to query for only organization
+        /// templates</param>
         /// <returns>The first scorecard template that satisfies the predicate or null if the predicate was null or no
         /// scorecard template satisfies the predicate</returns>
-        public async Task<ScorecardTemplateSummary> FindAsync(Func<ScorecardTemplateSummary, bool> predicate)
+        public async Task<ScorecardTemplateSummary> FindAsync(Func<ScorecardTemplateSummary, bool> predicate, string workspaceID=null)
```

Comment:
Spacing for all parameters assignment
```suggestion
        public async Task<ScorecardTemplateSummary> FindAsync(Func<ScorecardTemplateSummary, bool> predicate, string workspaceID = null)
```

Reply:
Updated